cli/cmd/deploy.go

```diff
@@ -106,23 +106,31 @@ var _deployCmd = &cobra.Command{
 				exit.Error(err)
 			}
 
+			local.OutputType = _flagOutput // Set output type for the Local package
 			deployResults, err = local.Deploy(env, configPath, projectFiles, _flagDeployDisallowPrompt)
 			if err != nil {
 				exit.Error(err)
 			}
 		}
 
-		if _flagOutput == flags.JSONOutputType {
+		switch _flagOutput {
+		case flags.JSONOutputType:
 			bytes, err := libjson.Marshal(deployResults)
 			if err != nil {
 				exit.Error(err)
 			}
 			fmt.Println(string(bytes))
 			return
+		case flags.MixedOutputType:
```

Comment:
Would it be better if we'd call it `base64-encoded-json` output type or something like this? What are your thoughts?

Reply:
It isn't exactly strictly base64-encoded-json, we can discuss this live.